SQL/activity/history.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
```

Comment:
The file name is not consistent with similar query files in other models.

Reply:
I can change the file names, but I think it's impossible to keep the previous ones. I think I should rename some old ones so as to make them look consistant in all models

Reply:
Why impossible? My recommendation is avoid being creative when naming common things. Inability to make assumptions about the code base hurts productivity greatly, especially if you're not the only person working on it.

Reply:
sure, I made some changes. I was living in the past when we had "register" for user :)))